codec_test.go

```diff
@@ -15,6 +15,7 @@ func BenchmarkEncoding(b *testing.B) {
 	// generate some fake data
 	data := generateRandData(128)
 	for codecName, codec := range codecs {
+		codec.Encode(data)
```

Comment:
I think I forgot to add a comment here to indicate that this is for making sure that the relevant encoder is cached

Reply:
Hmm, I wonder if that has any impact on the encoding speed for some codecs if it already ran on the same data 🤔 I'll add a comment and make sure that the init-run is with different data. Just in case.

Reply:
done in [9c35d77](https://github.com/celestiaorg/rsmt2d/pull/136/commits/9c35d7748872aeea319b4c4ba1ef880f404ce0c3)